### ISSUE TITLE
Add delete functionality for chapters

### ISSUE THREAD

**Issue by hypermystic** [hypermystic/bookit#6]

**Comment by savanipoojan78** [comment]
i would like to work on this issue

**Comment by guptapranav4299** [comment]
I would like to work on this issue @hypermystic

**Comment by hypermystic** [comment]
@guptapranav4299 yeah sure... assigned

**Comment by hypermystic** [comment]
Let me know if u face any issues in setting up things

**Comment by guptapranav4299** [comment]
I have setup the project but when I click on google signin option then nothing works after that...I have created the env file and mentioned the firebase config in the env file

**Comment by hypermystic** [comment]
@guptapranav4299 yeah...that is the one I am facing too... probably due to cors..so for the workaround instead of using the env files just directly add the firebase configuration values in the firebaseConfig.js file...and then u r good to go

**Comment by guptapranav4299** [comment]
still facing the same issue, after directly adding firebase configurations @hypermystic

**Comment by hypermystic** [comment]
@guptapranav4299 can you share me the screenshot of console

**Comment by guptapranav4299** [comment]
![image](https://user-images.githubusercontent.com/50111336/96410891-3f3c8780-1205-11eb-9d59-98d809585d40.png)
after this screen nothing occurs

**Comment by vaibhav111tandon** [comment]
@guptapranav4299 can you please also share the console logs if possible, because I am not able to reproduce this on my end

**Comment by vaibhav111tandon** [comment]
@hypermystic are you able to reproduce this
